environments.json

```diff
@@ -4936,7 +4936,74 @@
     "test_suites": [
       "es5",
       "es6",
-      "es2016plus"
+      "es2016plus",
+      "esnext",
+      "esintl",
+      "non-standard"
+    ]
+  },
+  "hermes0_8_1": {
+    "full": "Hermes 0.8.1",
+    "family": "Hermes",
+    "short": "Hermes 0.8.1",
+    "platformtype": "engine",
+    "release": "2021-07-12",
+    "obsolete": false,
```

Comment:
All these versions are receiving active support?  The retired versions should be `"obsolete": false`, the versions that are not in use anymore should be `"obsolete": "very"`.

Reply:
Cheers, just waiting for some feedback from Hermes on that.